app/views/drawings/_drawings.html.haml

```diff
@@ -1 +1,51 @@
-= render @drawings.desc
+.container
+  %h3 Manage Drawings
+  %table.table.table-bordered.table-condensed.table-hover.table-responsive
+    %thead
+      %tr
+        %th ID
+        %th File
+        %th Subject Matter
+        %th Description
+        %th Age
+        %th Artist's Gender
+        %th Story
+        %th Mood Rate
+        %th Edit
+        %th Delete
+        %th User Admin
+        %th Status
+    %tbody
+    - @drawings.each do |drawing|
+      %tr
+        %td
+          = drawing.id
+        %td
+          = image_tag drawing.image.url(:thumb)
+        %td
+          = drawing.subject_matter
+        %td
+          = drawing.description[0, 29].concat('...')
```

Comment:
We can create a helper method in the [drawings helper](https://github.com/empowerhack/DrawMyLife-Service/blob/master/app/helpers/drawings_helper.rb) to concatenate long text fields for this table and move logic out of the view. So we can call something like `formatted_table_text(drawing.some_attr)`and re-use it to concatenate Story down below as well (+ any long fields going forward)